src/System/Logging/LogSink/Core.hs

```diff
@@ -0,0 +1,56 @@
+module System.Logging.LogSink.Core (
+  Config(..)
+, Format
+, Sink
+, defaultFormat
+, stdErrSink
+, sysLogSink
+, mkLogSink
+) where
+
+import           Control.Monad
+import           System.IO
+import           System.Posix.Syslog
+import           System.Logging.Facade.Types
+import           System.Logging.Facade.Sink
+
+data Config = Config {
+  configFormat :: Format
+, configSinks :: [Sink]
+}
+
+type Format = LogRecord -> String
+
+type Sink = LogLevel -> String -> IO ()
+
+defaultFormat :: LogRecord -> String
+defaultFormat record = (shows level . location . showString ": " . showString message) ""
+  where
+    level = logRecordLevel record
+    mLocation = logRecordLocation record
+    message = logRecordMessage record
+    location = maybe (showString "") ((showString " " .) . formatLocation) mLocation
+
+    formatLocation :: Location -> ShowS
+    formatLocation loc = showString (locationFile loc) . colon . shows (locationLine loc) . colon . shows (locationColumn loc)
+      where colon = showString ":"
+
+sysLogSink :: Sink
+sysLogSink level = syslog (toPriority level)
+  where
+    toPriority :: LogLevel -> Priority
+    toPriority l = case l of
+      TRACE -> Debug
+      DEBUG -> Debug
+      INFO -> Info
+      WARN -> Warning
+      ERROR -> Error
+
+stdErrSink :: Sink
+stdErrSink _level = hPutStrLn stderr
```

Comment:
I think this is meant to be used in conjunction with a `Format` and `defaultFormat` does include the log level.